src/settings/MatchProfiles/ViewMatchProfile.js

```diff
@@ -218,22 +218,22 @@ export class ViewMatchProfile extends Component {
 
     // Here is mocked config file with mocked values, it should be replaced/rewritten once BE will be ready
     const formConfig = formConfigSamples.find(cfg => cfg.name === formName);
-    const matchDetails = get(matchProfile, ['matchDetails', '0'], []);
-    const {
-      incomingMatchExpression,
-      existingMatchExpression,
-    } = matchDetails;
-
-    const incomingFields = get(incomingMatchExpression, 'fields', []);
-    const existingFields = get(existingMatchExpression, 'fields', []);
-    const existingField = this.getValue(existingFields, 'field');
 
     const record = {
       ...matchProfile,
-      matchDetails: [{
-        incomingMatchExpression: {
-          fields: [
-            {
+      matchDetails: matchProfile.matchDetails.map(item => {
+        const {
+          incomingMatchExpression,
+          existingMatchExpression,
+        } = item;
```

Comment:
Such variable name like 'item' is meaningless for me. Consider renaming it.

Reply:
It is a variable inside a cycle that represents iterated item